pkg/rsyslog/rsyslog.conf

```diff
@@ -6,6 +6,8 @@ set $/IMGP = getenv("IMGP");
 
 module(load="imklog")
 module(load="imuxsock" SysSock.Name="/dev/log")
+module(load="imudp")
+input(type="imudp" port="514" address="127.0.0.1")
```

Comment:
This should be removed since we're not using this, right? @gkodali-zededa

Reply:
Done. I actually had this change in my local sandbox, but forgot to push yesterday.